test/resources/examples/generate.js

```diff
@@ -79,9 +79,9 @@ function generateTestCase(input, debug = false) {
     solutions: solutions.map(solution => {
       let objectiveValue
       try {
-        objectiveValue = solutionObjectiveValueComputation(orders, solution, true)
+        objectiveValue = solutionObjectiveValueComputation(orders, solution, strict)
       } catch (err) {
-        if (debug) {
+        if (strict && debug) {
```

Comment:
@nlordell  - Is this the problem you are referring to? Is this correct now? I was pretty sure this was already fixed as suggested.

Reply:
@bh2smith exactly what I meant!